scripts/install.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-packer_http=$(cat .packer_http)
+packer_http=$PACKER_HTTP_ADDR
```

Comment:
Why not just use `$PACKER_HTTP_ADDR` later in the script?

Separately - this means the `boot_command` can remove writing to the `.packer_http` file, right?

Reply:
Sometimes $PACKER_HTTP_ADDR is not corrtect, I think use the packer_http to control it. I only test it in virtualbox, so I could upgrade virtualbox boot_command.